package.json

```diff
@@ -47,6 +47,11 @@
     "vite-tsconfig-paths": "4.2.1",
     "vitest": "1.0.0-beta.1"
   },
+  "pnpm": {
+    "overrides": {
+      "react-dom": "18.2.0"
```

Comment:
why do we need to override react-dom here

Reply:
can we unify the version via other methods? override is not a good choice

Reply:
@chenjiahan use overrides like below or resolutions, will deduce the unmet peer problem and the react-dom 17.2.0 always will be installed , cannot limit the react-dom version, but the `react-json-view` `ansi-to-react` can use in react-dom v18 (https://github.com/nteract/ansi-to-react/issues/80)

`  "overrides": {
    "react-json-view": {
      "react": "18.2.0",
      "react-dom": "18.2.0"
    }
  },` 
  <img width="578" alt="image" src="https://github.com/user-attachments/assets/85a86b3a-9a49-42dc-8914-b32f8133c9df">

Reply:
Get, `react-json-view' seems to be a lack of maintenance.